electron-src/index.ts

```diff
@@ -439,6 +444,37 @@ ipcMain.handle(
   }
 );
 
+// TRELLO FUNCTIONS
+
+const getTrelloOptions = () => {
+  return {
+    key: process.env.NEXT_PUBLIC_TRELLO_KEY || "",
+    returnUrl: process.env.NEXT_PUBLIC_TRELLO_REDIRECT_URI || "",
+  };
+};
+
+ipcMain.on("trello:login", async () => {
+  const options = getTrelloOptions();
+  const trelloAuthUrl = getTrelloAuthUrl(options);
+
+  mainWindow?.loadURL(trelloAuthUrl);
+});
+
+ipcMain.handle(
+  "trello:get-profile-info",
+  async (event, accessToken: string) => {
+    const options = getTrelloOptions();
+
+    return await getMember(accessToken, options);
+  }
+);
+
+ipcMain.handle("trello:get-cards", async (event, accessToken: string) => {
+  const options = getTrelloOptions();
+
+  return await getCardsOfMember(accessToken, options);
+});
+
```

Comment:
Again as suggestion - can we move this code to separate trello api file to make our main a little bit less?

Reply:
idk how it'll implement correctly, we discussed it with you